src/content/cloud/registry.mdx

```diff
@@ -50,3 +50,27 @@ Any image pushed into the Okteto Registry is private. You'll need to authenticat
 Namespaces in Okteto Cloud are automatically allowed to pull images that belong to their namespace automatically. If your application uses images from the Okteto Registry, it'll be able to pull container images without any extra configuration.
 
 To pull an image from a different namespace, you'll need to create and configure the required `imagePullSecrets` as outlined [here](/docs/reference/faqs/#how-to-use-private-images).
+
+## Push helm chart into the Okteto Registry
+
+Just as you can push images to the Okteto Registry, you also have the option to push Helm charts. To do this, the first step is to log in to the registry where you want to push the chart:
+
+```
+helm registry login registry.cloud.okteto.net -u my-username -p my-password
+```
+
+You can get your credentials needed to log in with Helm against the Okteto Registry by running the `okteto context show` command. With it you can retrieve your username, password (token) and the domain from your registry.
+
+Once you are logged against the Okteto Registry you can push the packaged chart to the Okteto Registry by using the command below:
```

Comment:
```suggestion
Once you are logged in you can push any packaged chart to the Okteto Registry by using the command below:
```

Reply:
I think this should be step 1 of the instructions:
1. Set your context
2. Get the values using okteto context show
3. Log in to registry with the helm cli
4. Push chart
5. Pull chart